### ISSUE TITLE
What error and how can to fix? Thanks

### ISSUE THREAD

**Issue by achoe** [Pr0x13/iBrutr#2]
![screen shot 2015-10-02 at 16 42 37](https://cloud.githubusercontent.com/assets/6562187/10243692/9f98472a-6924-11e5-8eeb-e319819bd69f.png)
Hello bro; longtime not see you.
After run in longtime this says that. So how can to fix that and one thing more in many option which one will get result because i tried with mine this says: "EEk! We ran out password" with wordlist.lst only for 3 words.
And Anywhere can make wordlist with requirement of Apple on crunch. Any suggest please/
So anyway thanks for your hard work. I hope first your product i can use for paid. 
Hope u have full health; lucky to continue with any exploit. 
Goodbye and hope can get your reply soon.

**Comment by modserv** [comment]
About the error, modify the memory_limit option in php.ini file. Your current memory_limit is 128M, try and make it 256M or 512M.

**Comment by achoe** [comment]
ok thanks @modserv and i fixed that by add one line below that line 121
ini_set('memory_limit', '-1');
and this worked very well. Anyone in there got done from this code? PLease share experience. 
I'm trying to fins another wordlist with apple requirement; i hope this will work well if wordlist perfect.
Anyway thanks!

**Comment by modserv** [comment]
Tried it on my account, it didn't crack the password but it got locked.

**Comment by achoe** [comment]
lol same here

**Comment by modserv** [comment]
Did you have it unlocked @achoe ?

**Comment by achoe** [comment]
not yet bro. I tried on my acc and this not get "This is password" 
But i must unlocked this last night didn't know reason. LOL
Did u see anyone get unlocked from this?

**Comment by modserv** [comment]
Yes, I unlocked it successfully, just proceed with Forgot your password and you will get this email, click the link and choose Unlock.
![2015-10-05_2-01-40](https://cloud.githubusercontent.com/assets/3301963/10271073/1169c932-6b05-11e5-81a2-61470efd7db0.jpg)

**Comment by achoe** [comment]
yes i know, i mean did u see anyone use iBrutr to unlocked their phone?

**Comment by modserv** [comment]
Actually, no lol.
